lib/fluent/supervisor.rb

```diff
@@ -439,14 +439,17 @@ def show_plugin_config
     def supervise
       $log.info "starting fluentd-#{Fluent::VERSION}"
 
+      rubyopt = ENV["RUBYOPT"]
       if Fluent.windows?
         fluentd_spawn_cmd = ServerEngine.ruby_bin_path + " -Eascii-8bit:ascii-8bit "
+        fluentd_spawn_cmd << ' "' + rubyopt.gsub('"', '""') + '" ' if rubyopt
```

Comment:
I cannot understand the reason why this code is like that.
Please add code comment to describe it.